### PR TITLE
chore: sync updates from main into v11

### PR DESCRIPTION
**Pull request by alisonjoseph**

### CODE REVIEW

File: src/components/TypesetStyle/TypesetStyle.js [hunk 1]
```diff
@@ -13,11 +13,888 @@ import InputRange from './InputRange';
 import StickyContainer from './StickyContainer';
 import TypesetExample from './TypesetExample';
 
-import typeScale from './typeScale';
-import typeSets from './typeSets';
-
 const { prefix } = settings;
 
+const typeScale = {
```
Comment (aagonzales) [inline src/components/TypesetStyle/TypesetStyle.js:18]: We don't want to important any of the v10 type docs to v11. We have v11 updates already on the v11 site so don't include any of this in the PR.

File: src/pages/components/button/usage.mdx [hunk 2]
```diff
@@ -228,12 +228,11 @@ size. For guidance on button heights as well as type sizes and tokens, see
...
-| Large (expressive) | The larger expressive type size within this button provides balance when used with 16px body copy. Used by the IBM.com team in website banners.                                                                                                          |
-| Extra large        | Use when buttons bleed to the edge of a larger component, like side panels or modals. _Note: This variant is not offered in the kit because it does not have a coded counterpart. Carbon only offers this button in the context of the modal component._ |
-| 2XL                | Use when buttons bleed to the edge of a larger component, like side panels or modals. _Note: This variant is not offered in the kit because it does not have a coded counterpart. Carbon only offers this button in the context of the modal component._ |
+| _Full bleed_       | Use when buttons bleed to the edge of a larger component, like side panels or modals. _Note: This variant is not offered in the kit because it does not have a coded counterpart. Carbon only offers this button in the context of the modal component._ |
```
Comment (aagonzales) [inline src/pages/components/button/usage.mdx:231]: This is the old v10 content, do not include

File: src/pages/components/content-switcher/style.mdx [hunk 3]
```diff
@@ -13,26 +13,25 @@ Content switchers have two main states: `selected` and `unselected`. By default,
...
-| Class                       | Property         | Color token               |
-| --------------------------- | ---------------- | ------------------------- |
-| `.bx--content-switcher-btn` | background-color | `$background`             |
-| `.bx--content-switcher-btn` | text color       | `$text-secondary`         |
-| `--selected`                | background-color | `$layer-selected-inverse` |
-| `--selected`                | text color       | `$text-inverse`           |
-| `:after`                    | divider          | `$border-subtle`          |
+| Class                       | Property         | Color token      |
+| --------------------------- | ---------------- | ---------------- |
+| `.bx--content-switcher-btn` | background-color | `$ui-background` |
```
Comment (aagonzales) [inline src/pages/components/content-switcher/style.mdx:18]: This is v10 old content, do not include.

File: src/pages/components/content-switcher/style.mdx [hunk 4]
```diff
@@ -44,13 +43,13 @@ Hover states only apply to `unselected` buttons.
...
-word in a phrase and any proper nouns capitalized. The label text should not
-exceed three words.
+Content switcher text should be set in sentence case, with only the first word
+in a phrase and any proper nouns capitalized. The text should not exceed three
+words.
 
-| Element | Font-size (px/rem) | Font-weight   | Type token         |
-| ------- | ------------------ | ------------- | ------------------ |
-| Label   | 14 / 0.875         | Regular / 400 | `$body-compact-01` |
+| Class                       | Font-size (px/rem) | Font-weight   | Type style       |
```
Comment (aagonzales) [inline src/pages/components/content-switcher/style.mdx:50]: This is old content, do not include.

File: src/pages/components/data-table/usage.mdx [hunk 5]
```diff
@@ -358,8 +358,8 @@ import {
 
 ### Sizing
 
-The data table is available in five different row sizes: extra large, large,
-medium, small, extra small.
+The data table is available in four different
```
Comment (aagonzales) [inline src/pages/components/data-table/usage.mdx:361]: This is old content, do not include.

**Review by aagonzales — COMMENTED** [review]
Ignore "updates" from the following component usage and style docs pages, content is already correct on the v11 site and this is importing old content. 
- Button
- Content switcher 
- data tables
- dropdown
- modal
- Notification 
- Tabs
- tooltips

**Review by aagonzales — APPROVED** [review]
Yeah this looks correct now!

**Review by aledavila — APPROVED** [review]

**Pull request merged**